pkg/solver/server.go

```diff
@@ -345,6 +348,12 @@ func (server *solverServer) addJobOffer(jobOffer data.JobOffer, res corehttp.Res
 		return nil, fmt.Errorf("job creator address does not match signer address")
 	}
 
+	versionHeader, _ := http.GetVersionFromHeaders(req)
+	minVersion, ok := server.versionConfig.IsSupported(versionHeader)
+	if !ok {
+		return nil, fmt.Errorf("Please update to minimum supported version %s or newer: https://github.com/Lilypad-Tech/lilypad/releases", minVersion)
```

Comment:
Let's add a log statement here too

Reply:
Yeah good call! Added in db9caa2b0b0609898642732b33941014cba8b448